mod-orders-storage/schemas/contributor.json

```diff
@@ -7,11 +7,14 @@
       "description": "the name of a contributor to the material",
       "type": "string"
     },
-    "contributorType": {
+    "contributorNameType": {
```

Comment:
Please indicate that this is id by adding `Id` suffix

Reply:
Done